src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -1,10 +1,119 @@
+import { Box, Grid, Theme, Typography, useTheme } from '@mui/material';
+import TextField from '@terraware/web-components/components/Textfield/Textfield';
+import { useEffect, useRef, useState } from 'react';
+import strings from 'src/strings';
 import { ServerOrganization } from 'src/types/Organization';
+import PageHeaderWrapper from '../common/PageHeaderWrapper';
 import TfMain from '../common/TfMain';
+import { makeStyles } from '@mui/styles';
+import { Table, TableColumnType } from '@terraware/web-components';
```

Comment:
similarly for `src/components/common/table/index.tsx` and `types.ts` 🤷 Again, I prefer what you have here.